src/lib.rs

```diff
@@ -52,18 +84,61 @@ impl<T: 'static + Send + Sync> Default for DefaultRaycastingPlugin<T> {
 pub enum RaycastSystem {
     BuildRays,
     UpdateRaycast,
-    UpdateDebugCursor,
+    #[cfg(feature = "debug")]
+    AddOrRemoveDebugCursors,
+    #[cfg(feature = "debug")]
+    EnableOrDisableDebugCursors,
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum Enabled {
```

Comment:
I'd prefer to name this something like `ActiveState` or something. Then the `PluginState` would be something like
```rust
if plugin.active == ActiveState::Enabled {
// ...
```

Reply:
Done